custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
+
+migration_query = """
+UPDATE {new_table} B
+SET
+B.pse_daily_attendance = A.pse_days_attended,
+B.pse_daily_attendance_male = CASE WHEN B.sex = 'M' THEN A.pse_days_attended ELSE NULL END,
+B.pse_daily_attendance_female = CASE WHEN B.sex = 'F' THEN A.pse_days_attended ELSE NULL END
+FROM {old_table} A
+WHERE A.doc_id = B.doc_id and A.month_start = B.month_start
```

Comment:
I don't think so. repeat_iteration is what determines the month. 

So if one of the data sources had processed a case last month and the other data source processed one this month (I don' thtink this will happen), then it could be that in one data source 1 refers to june 2017 and the other 1 refers to july 2017.

May have been better to have a primary key on doc_id, month instead, but I think it's required to have repeat_iteration in UCRs if you use an iterator. would need to look more into it to confirm